shunting-yard-replicator/src/main/java/com/hotels/shunting/yard/replicator/exec/messaging/MessageReaderAdapter.java

```diff
@@ -65,6 +71,10 @@ private MetaStoreEvent map(ListenerEvent listenerEvent) {
 
     EventType eventType = listenerEvent.getEventType();
 
+    if (!tableSelector.canProcess(listenerEvent)) {
+      return builder.build();
```

Comment:
This still returns an event for the table we don't want to process. How is this going to prevent the table replication from happening?